releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,24 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 7.0.0 - November 21, 2023
+### Breaking Changes
+* Removed .NET Core 3.1 as a target framework as .NET Core 3.1 is out of support. Updated `Microsoft.Extensions` dependencies to version 6.0.0 and later. [#482](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/482)
+* `AddAzureAppConfiguration` now throws errors for invalid inputs when the `optional` flag is equal to `true`. [#408](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/408)
+
+### Enhancements
+* This is the first stable release of the following API introduced in the 7.0.0-preview release. [#422](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/422)
+
+   ```cs
+   public AzureAppConfigurationOptions SelectSnapshot(string name)
+   ```
+* Added support for the `requirement_type` property for feature flags introduced in version 2.6.0-preview of the Microsoft.FeatureManagement library. [#475](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/475)
+* The provider will now continuously retry when loading data from App Configuration on startup until either the load is successful or the timeout value in `AzureAppConfigurationOptions.StartupOptions.Timeout` has elapsed. `AzureAppConfigurationOptions.StartupOptions` can be set using the following new API. [#488](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/488)
```

Comment:
Besides what, it's important to explain why, so customers understand why this change is relevant to them at all. Suggest below:

The initial configuration loading is changed to use time-based retry from count-based retry in case of failures. The time-based retry often works more effectively in real-world scenarios, making applications more resilient to transient errors during startup. By default, the retry timeout is set to 100 seconds, but you can customize it using the following new API.

(please link to the issue instead of the PR)